service_container.rst

```diff
@@ -133,9 +147,15 @@ the service container *how* to instantiate it:
 
         # app/config/services.yml
         services:
-            app.message_generator:
-                class:     AppBundle\Service\MessageGenerator
-                arguments: []
+            # default configuration for services in *this* file
+            _defaults:
+                autowire: true
+                autoconfigure: true
+                public: false
+
+            # loads services from whatever directories you want (you can update this!)
+            AppBundle\:
+                resource: '../../src/AppBundle/{Service,Command,Form,EventSubscriber,Twig,Security}'
```

Comment:
I don't see a big gain, that makes one more thing to learn for newcomers while relative paths are understandable by everyone.